Index.html

```diff
@@ -0,0 +1,215 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Chainpal code</title>
+        <link href="./styled.css" rel="stylesheet">
+
+    </head>
+    <body>
+        <div class="wrapper">
+            <section class="bannerSection">
+                <div class="container">
+                    <h3 class="h3CustomMobile">
+                        Join the Escrow Manager Program
+                    </h3>
+                    <div class="innerWrapper">
+
+                        <div class="leftDiv">
+                            <div class="iamgeDiv">
+                                <img class="imageDesign"
+                                    src="https://www.chainpals.io/assets/images/guest/png/become-escrow-manager.png">
+                            </div>
+
+                        </div>
+                        <div class="rightDiv">
+                            <div class="rightDivContainer">
+                                <h3 class="h3Custom">
+                                    Join the Escrow Manager Program
+                                </h3>
+                                <p class="description">
+                                    Become a Chainpals Escrow Manager and start
+                                    earning passive income by hodling EM NFT.
+                                </p>
+
+                                <div class="button">
+                                    <a>LEARN MORE</a>
+                                </div>
+
+                            </div>
+
+                        </div>
+                    </div>
+                </div>
+                </section class="roadMapSection">
+                <!-- --------next section---------- -->
+                <section>
+                    <div class="roadMApContainer">
+                        <p class="roadMapTitle">ROAD MAP</p>
+                    </div>
+                    <div class="backgroundImagefeature">
+                        <img src="https://www.chainpals.io/assets/images/guest/svgs/roadmap.svg" class="secondimg">
+                    </div>
+                    <div class="roadMApWrapper">
+                        <div class="swiperWrapper">
+                            <div class="swiperWrapperText">
+                                <div class="timeStampText">
+                                    <span class="date">Q4 2021</span>
```

Comment:
date >> roadmapDate 

please don't use preserved variable in class name @PRABgit01